pom.xml

```diff
@@ -7,7 +7,7 @@
   <version>0.1-SNAPSHOT</version>
   <properties>
     <compiler-plugin.version>3.8.1</compiler-plugin.version>
-    <maven.compiler.release>11</maven.compiler.release>
+    <maven.compiler.release>17</maven.compiler.release>
```

Comment:
Maybe just put this in a different commit, can still be in the PR

Reply:
Some PR was merged and updated to 17.